### ISSUE TITLE
Add the choice of copying or not files during import

### ISSUE THREAD

**Issue by A-d-r-i** [tenacityteam/tenacity-legacy#454]
- [x] I have read the specified guidelines for issues

### **Is your feature request related to a problem? Please describe.**
Since version [2.3.3](https://github.com/audacity/audacity/releases/tag/Audacity-2.3.3) audacity has removed a very useful feature: the creation (or not) of a copy of sound when imported. **In the release**: "Removed On-Demand aliased files (copy files instead)".

➔ The last version where the functionality is present is therefore 2.3.2: https://github.com/audacity/audacity/releases/tag/Audacity-2.3.2

### **Describe the solution you'd like**
I would like to be able to find the popup window (or at least the parameter in the options) in order to be able **to refuse the copy of the sound file during the import**. I load large and numerous files, the copy takes a monstrous time.

### **Describe alternatives you've considered**
Disable automatic file copying during the import?

### **Additional context**
**The window that appeared during the import:**
![image](https://user-images.githubusercontent.com/27277698/128830879-0f1de4bd-261f-4887-85f4-e2f0a3262b7e.png)

**The parameter in the options:**
![image](https://user-images.githubusercontent.com/27277698/128831239-bfb129c9-0724-47b5-a5c5-3d38cebe3634.png)

**Comment by n0toose** [comment]
Spotted the commit here: 65682e7a0c9469379a1a5e76ac264b0e493cef62

**Comment by n0toose** [comment]
The reason why Audacity removed this feature is mentioned here: https://wiki.audacityteam.org/wiki/New_features_in_Audacity_2.3.3#noleol

"Normalizing or alternatively compression should usually only be done as one of the last steps in producing audio, for example if one knows the audio is destined to go into an MP3 file or CD."

**Comment by A-d-r-i** [comment]
> "Normalizing or alternatively compression should usually only be done as one of the last steps in producing audio, for example if one knows the audio is destined to go into an MP3 file or CD."

Thanks! But be careful, I am not talking about this "Normalize on Load feature" feature, but the "On demand loading of aliases files" (https://wiki.audacityteam.org/wiki/New_features_in_Audacity_2.3.3#odleol) functionality.

I understand the reasons for the deletion but I would very much like to see the option return 😊

**Comment by Paul-Licameli** [comment]
To set history straight, what @panos cites is not relevant.  The feature that was removed was something else, a use of a pool of background threads to load the referenced imported files on demand, not importing all the data up front and pausing.

See obsolete code in src/on demand.

The implementation was not easily compatible with the major change of project file format in version 3, and so the feature was abandoned.  The “unitary” file format was considered much more important.

**Comment by n0toose** [comment]
Yeah, I got that, I just tried to skim through for a commit in bulk and tried to search through the tag, found something that corresponded to the contents of the screenshot and just straight up pasted it. Could've done better there. Thanks for the context (as well as for generally being so amiable), @Paul-Licameli.
